scripts/update-release-branch.sh

```diff
@@ -10,19 +10,18 @@ VERSION=`echo "$1.0"`
 echo "Updating branch to ${BRANCH}..."
 
 # Replace source links in docs and examples
-find docs -iname "*.md" -type f -exec sed -i '' -E "s/react-map-gl\/(tree|blob)\/(master|[0-9\.]+-release)/react-map-gl\/tree\/${BRANCH}/g" {} \;
-find examples -maxdepth 0 -iname "*.md" -type f -exec sed -i '' -E "s/react-map-gl\/(tree|blob)\/(master|[0-9\.]+-release)/react-map-gl\/tree\/${BRANCH}/g" {} \;
-find examples/*/src -iname "*.js" -type f -exec sed -i '' -E "s/react-map-gl\/(tree|blob)\/(master|[0-9\.]+-release)/react-map-gl\/tree\/${BRANCH}/g" {} \;
+find docs -mindepth 2 -iname "*.md" -exec perl -i -pe "s/react-map-gl\/(tree|blob)\/(master|[0-9\.]+-release)/react-map-gl\/tree\/${BRANCH}/g" {} \;
```

Comment:
I'm not super confident in adding `-mindepth 2` here. The reason that I added it is that there are a few markdown files under /docs (README.md, upgrade-guide.md, and whats-new.md) that seem to be manually maintained. I’d definitely appreciate any additional thoughts on this.